src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -242,6 +245,32 @@ def is_running(self) -> bool:
             cluster_name=self._k3d_cluster_name
         )
 
+    def list_manual_setup_steps(
+        self, container_registry_name: str, container_registry_path: str
+    ) -> None:
+        """Logs manual steps needed to setup the Kubeflow local orchestrator."""
+        global_config_dir_path = zenml.io.utils.get_global_config_directory()
+        logger.error("Unable to spin up local Kubeflow Pipelines deployment.")
+        logger.info(
+            "If you wish to spin up this Kubeflow local orchestrator manually, "
+            "please enter the following commands (substituting where appropriate):\n"
+        )
+        logger.info(
+            f"k3d cluster create CLUSTER_NAME --registry-create {container_registry_name} --registry-config {container_registry_path} --volume {global_config_dir_path}:{global_config_dir_path}"
+        )
+        logger.info(
+            f"kubectl --context CLUSTER_NAME apply -k github.com/kubeflow/pipelines/manifests/kustomize/env/platform-agnostic-pns?ref={KFP_VERSION}"
+        )
+        logger.info(
+            "kubectl --context CLUSTER_NAME wait --timeout=60s --for condition=established crd/applications.app.k8s.io"
+        )
+        logger.info(
+            f"kubectl --context CLUSTER_NAME apply -k github.com/kubeflow/pipelines/manifests/kustomize/env/platform-agnostic-pns?ref={KFP_VERSION}"
+        )
+        logger.info(
+            f"kubectl --namespace kubeflow port-forward svc/ml-pipeline-ui {self.kubeflow_pipelines_ui_port}:80"
+        )
+
```

Comment:
nit: this would look better as a multi-line string, or even a list of strings that you can join later. Unless there's a reason why you'd prefer having each line logged separately.

Reply:
thanks for considering my suggestion, the code looks cleaner this way